src/App.js

```diff
@@ -1,24 +1,20 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import {useState} from "react"
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import Header from "./Components/Header";
+import ReviewList from "./Components/ReviewList"
+
+// import ReviewCard from "./Components/ReviewCard";
 
 function App() {
+  const [reviews, setReviews] = useState(null)
```

Comment:
you could have this state in ReviewList instead of app, having it as low as possible but as high as necessary and inside ReviewList it can be passed as props to ReviewCard